cmake/onnxruntime_mlas.cmake

```diff
@@ -164,6 +164,20 @@ else()
     )
   elseif(X86_64)
     enable_language(ASM)
+    
+    set (additional_compile_flags "")
```

Comment:
Instead of this, can you just do:

set(CMAKE_ASM_FLAGS "${CMAKE_ASM_FLAGS} -the_flags_to_use")

I think you can also then just do the below to pick up whatever flag would have been used for C++ code:

set(CMAKE_ASM_FLAGS "${CMAKE_ASM_FLAGS} {CMAKE_CXX_OSX_DEPLOYMENT_TARGET_FLAG}")

This change could go in CMakeLists.txt too if you want it to be more global.

Reply:
I see the warnings back with the proposed change.

Reply:
Resolved the warnings based on your new offline proposal, thanks!